Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Menu/Menu.bind

```diff
@@ -24,112 +136,70 @@
-          <MenuFlyoutItem Text="Go To All" />
-        </MenuFlyoutSubItem>
-
-        <MenuFlyoutSubItem Text="Find and Replace">
-          <MenuFlyoutItem Text="Find" />
-          <MenuFlyoutItem Text="Replace" />
-        </MenuFlyoutSubItem>
-      </controls:MenuItem>
-
-      <controls:MenuItem controls:Menu.InputGestureText="Alt+V"
-                         Header="^View">
-        <MenuFlyoutSubItem Text="Designer">
-          <MenuFlyoutItem Text="VS Designer" />
-          <MenuFlyoutItem Text="Blend" />
-        </MenuFlyoutSubItem>
-
-        <MenuFlyoutSubItem Text="Solution Explorer">
-          <MenuFlyoutItem Text="Solutions" />
-          <MenuFlyoutItem Text="Projects" />
-        </MenuFlyoutSubItem>
-      </controls:MenuItem>
-
-      <controls:MenuItem controls:Menu.InputGestureText="Alt+P"
-                         Header="^Project">
-        <MenuFlyoutSubItem Text="Add Class">
-          <MenuFlyoutItem Text="Class" />
-          <MenuFlyoutItem Text="Interface" />
-        </MenuFlyoutSubItem>
-
-        <MenuFlyoutSubItem Text="Add new data source">
-          <MenuFlyoutItem Text="SQL" />
-          <MenuFlyoutItem Text="NoSQL" />
-        </MenuFlyoutSubItem>
+                        Text="Exit"
+                        Margin="0,0,0,8" />
       </controls:MenuItem>
 
-      <controls:MenuItem controls:Menu.InputGestureText="Alt+B"
-                         Header="^Build">
-        <MenuFlyoutItem Text="Build Solution" />
-        <MenuFlyoutItem Text="Rebuild Solution" />
+      <controls:MenuItem controls:Menu.InputGestureText="Alt+E" Header="^Edit">
+        <MenuFlyoutItem controls:Menu.InputGestureText="Ctrl+X"
+                        Command="{StaticResource GenericCommand}"
+                        CommandParameter="Cut"
+                        Text="Cut"
+                        Icon="Cut" />
+        <MenuFlyoutItem controls:Menu.InputGestureText="Ctrl+C"
+                        Command="{StaticResource GenericCommand}"
+                        CommandParameter="Copied"
+                        Text="Copy"
+                        Icon="Copy" />
+        <MenuFlyoutItem controls:Menu.InputGestureText="Ctrl+V"
+                        Command="{StaticResource GenericCommand}"
+                        CommandParameter="Pasted"
+                        Text="Paste"
+                        Icon="Paste" />
       </controls:MenuItem>
 
-      <controls:MenuItem controls:Menu.InputGestureText="Alt+D"
-                         Header="^Debug">
-        <MenuFlyoutSubItem Text="Windows">
-          <MenuFlyoutItem Text="Windows 8" />
-          <MenuFlyoutItem Text="Windows 10" />
-        </MenuFlyoutSubItem>
-
-        <MenuFlyoutSubItem Text="Graphics">
-          <MenuFlyoutItem Text="Canvas" />
-          <MenuFlyoutItem Text="Grid" />
-        </MenuFlyoutSubItem>
-      </controls:MenuItem>
+      <controls:MenuItem  Name="FileMenu" controls:Menu.InputGestureText="Alt+V" Header="^View" />
```

Comment:
This should have items as otherwise it just pops-down an empty drop-down or it should act as a button?